src/utils/updateDatabase.ts

```diff
@@ -15,23 +15,8 @@ export async function updateDatabase(collections: Collection[]) {
 
   try {
     if (connection.isConnected) {
-      for (const entry of collections) {
-        if (!entry.price_guidance) {
-          try {
-            const priceGuidance = await getPriceGuidance(entry.slug)
-            extend(entry, { price_guidance: priceGuidance })
-          } catch (e) {
-            console.log(
-              "[PriceGuidance] Unable to set price guidance on " + entry.slug
-            )
-            console.log(e.message)
-          }
-        }
-        await collection.update({ slug: entry.slug }, entry, { upsert: true })
-        console.log("Successfully updated: ", entry.slug, entry.title)
-      }
-
-      console.log("Successfully updated collections database")
+      await upsert_data(collection, collections)
```

Comment:
This is an unfortunate naming similarity. Is there any renaming we could do to make it clear that one is a collection in the mongo sense, and one is a set of collections in the sense of our business model?